src/index.html

```diff
@@ -1,13 +1,454 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>THE MET</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="./images/icons/favicon.ico"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a
+              href="#"
+              alt="THE MET logo"
+              class="top-bar__logo"
+            >
+            </a>
+            <div class="top-bar__icons">
+              <a href="tel: +1 212-535-7710" class="icon icon--phone">
```

Comment:
Need to fix format phone only in href everywhere
```suggestion
              <a href="tel:12125357710" class="icon icon--phone">
```